### ISSUE TITLE
Trouble getting workflow to work on new macbook

### ISSUE THREAD

**Issue by Dr4hcir** [alexchantastic/alfred-lipsum-workflow#9]
Hi, I'm new to github so excuse me if I'm in the wrong area to get this support. I recently moved to a new Macbook and migrated the Alfred workflow manually through export and import. The Lorem Ipsum workflow doesn't work. At first there were some errors like "python: command not found". When googling I saw possible issues with PHP not working perfectly on new MacOS installations. I first downloaded the newest version (4.0.2) and got another error message:

```
[11:20:36.299] Lorem Ipsum[Keyword] Processing complete
[11:20:36.302] Lorem Ipsum[Keyword] Passing output '5' to Run Script
[11:20:36.319] ERROR: Lorem Ipsum[Run Script] sh: lipsum.sh: No such file or directory
[11:20:36.325] Lorem Ipsum[Run Script] Processing complete
[11:20:36.325] Lorem Ipsum[Run Script] Passing output '' to Copy to Clipboard
```

I assume I did something wrong somewhere, so I deleted the script and installed the new version again, restarted Alfred, but I get the same message in the debug log. Any idea what is wrong?

**Comment by alexchantastic** [comment]
Thanks for the detailed bug report and for bringing this to my attention!

It seems like I somehow did not package `lipsum.sh` into the workflow file when creating v4.0.2. I've created a new release ([v4.0.3](https://github.com/alexchantastic/alfred-lipsum-workflow/releases/tag/v4.0.3)) which correctly includes the script.

Please try installing the latest version and let me know if that works for you.

**Comment by Dr4hcir** [comment]
@alexchantastic  Solved with installing the new version 4.0.3. Thank you!!

**Comment by Dr4hcir** [comment]
issue solved with new version. Closed
